en/SUMMARY.md

```diff
@@ -132,21 +132,13 @@
 
   - [Flight Controllers (Autopilots)](flight_controller/index.md)
     - [Flight Controller Selection](getting_started/flight_controller_selection.md)
-    - [Mounting the Flight Controller](assembly/mount_and_orient_controller.md)
```

Comment:
Unrelated change. Not completely certain about this, but it feels odd to have all the stuff about how you mount and use the flight controller before the list of what is available.

I could have moved the FC selection topic down, but I think this is is better. Matches the index structure now too.